spacy/util.py

```diff
@@ -993,9 +996,17 @@ def run_command(
     except FileNotFoundError:
         # Indicates the *command* wasn't found, it's an error before the command
         # is run.
+
+        if tool is None:
+            # On Windows we may be unable to get the executable name.
+            tool_check = ""
+        else:
+            tool_check = f" Do you have {tool} installed?"
```

Comment:
This is not ideal but I'm not sure what we could do that's better. `shlex.split` would probably work 90% of the time, and it wouldn't be unsafe because the value of `tool` here is just used for debugging purposes.